scripts/devtools/tpl-manager.py

```diff
@@ -89,26 +89,13 @@ def parse_spec_list(file_path):
 #------------------------------
 # Dependencies
 #------------------------------
-def build_deps(args):
+def build_spack(args):
   print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
-  print("~~~~~ Building Dependencies")
+  print("~~~~~ Building and Configuring Spack")
```

Comment:
We aren't necessarily building spack, just configuring it
```suggestion
  print("~~~~~ Configuring Spack")
```